clientv3/watch.go

```diff
@@ -462,6 +471,9 @@ func (w *watchGrpcStream) run() {
 					close(ws.recvc)
 					closing[ws] = struct{}{}
 				}
+			case pbresp.PermissionDenied:
```

Comment:
this should be handled through the `pbresp.Created` path if it's only going to happen on create events; it should work as-is if returning a createresponse with canceled=true and id=-1.